zerver/tests/test_signup.py

```diff
@@ -176,7 +176,7 @@ def test_register(self):
         with queries_captured() as queries:
             self.register("test@zulip.com", "test")
         # Ensure the number of queries we make is not O(streams)
-        self.assert_max_length(queries, 69)
+        self.assert_max_length(queries, 88)
```

Comment:
Why did you change this?  It seems likely that adding 19 new database queries would be caused by a bug in your changes...